### DISCUSSION TITLE
"E: You don't have enough free space in /var/cache/apt/archives/" on ER-X

### DISCUSSION THREAD

**Discussion by luna-xenia** [jamesog/tailscale-edgeos#8] [General]
I am trying to install Tailscale, but I don't think my router has enough space. I have tried `apt-get autoclean` and `apt-get clean`, but I still don't have enough. Here is the output of `df- h`:
```
ubi0_0                  214.9M    199.1M     11.1M  95% /root.dev
overlay                 214.9M    199.1M     11.1M  95% /
devtmpfs                122.9M         0    122.9M   0% /dev
tmpfs                   123.6M         0    123.6M   0% /dev/shm
tmpfs                   123.6M    360.0K    123.3M   0% /run
tmpfs                     5.0M         0      5.0M   0% /run/lock
tmpfs                   123.6M         0    123.6M   0% /sys/fs/cgroup
tmpfs                   123.6M     12.0K    123.6M   0% /tmp
tmpfs                   123.6M         0    123.6M   0% /run/shm
tmpfs                   123.6M         0    123.6M   0% /lib/init/rw
tmpfs                   123.6M      3.7M    119.9M   3% /var/log
none                    123.6M    540.0K    123.1M   0% /opt/vyatta/config
tmpfs                    24.7M         0     24.7M   0% /run/user/1001
```

I was wondering if anyone encountered the same issue and how they fixed it.

**Comment by jamesog** [comment]
@slipdip123 I've converted this to a discussion as an issue isn't really the right place. Maybe people are more likely to respond here.

**Comment by luna-xenia** [comment]
Sorry about that! Haven’t used GitHub in a while

**Comment by milkcocoa** [comment]
@slipdip123 Hi! Did you delete old system images? 

As the following manual mentions, how about deleting old ones with `delete system image` command?
https://help.ui.com/hc/en-us/articles/205146110-EdgeRouter-How-to-Upgrade-the-EdgeOS-Firmware

**Reply by luna-xenia** [reply]
This worked! Thank you. Unfortunately it isn't letting me mark your answer as correct

**Reply by abetancort** [reply]
Remember to uninstall tailscale when upgrading to a new firmware or the process will fail due to a lack of free space to create the backup image of your current firmware.
